test/functional/web_api_controller_test.rb

```diff
@@ -0,0 +1,16 @@
+# def index
```

Comment:
Also if you could delete these comments, that'd be great. Once this gets sorted out, this looks like a great PR - pretty self contained functionally, thank you!